pandas/core/frame.py

```diff
@@ -3877,17 +3884,38 @@ def nlargest(self, n, columns, keep='first'):
         3  10  c  3.0
         4  -1  e  4.0
 
+        In the following example, we will use ``nlargest`` to select the three
+        rows having the largest values in column "a".
+
         >>> df.nlargest(3, 'a')
             a  b    c
         1  10  b  2.0
         3  10  c  3.0
         2   8  d  NaN
 
+        When using ``keep='last'``, ties are resolved in reverse order:
+
         >>> df.nlargest(3, 'a', keep='last')
             a  b    c
         3  10  c  3.0
         1  10  b  2.0
         2   8  d  NaN
+        
+        To order by the largest values in column "a" and then "c", we can
+        specify multiple columns like in the next example.
+
+        >>> df.nlargest(3, ['a', 'c'])
+            a  b    c
+        3  10  c  3.0
+        1  10  b  2.0
+        2   8  d  NaN
+
+        The dtype of column "b" is `object` and attempting to get its largest
```

Comment:
Just to generalize could instead say "Attempting to use ``nlargest`` on an non-numeric dtypes will raise a ``TypeError``"